processor/src/lib.rs

```diff
@@ -138,6 +138,7 @@ where
     A: AdviceProvider,
 {
     system: System,
+    kernel: Kernel,
```

Comment:
If we need to keep track of `kernel` for a process, it should be located in `KernelRom` chiplet rather than here.

Reply:
Fixed!